distribution/lib/Standard/Base/0.0.0-dev/src/Data/Ordering.enso

```diff
@@ -27,7 +27,7 @@ from project.Data.Boolean import all
 
     ```
     type Comparator T
-        compare : T -> T -> (Ordering|Nothing)
+        compare : T -> T -> (Ordering|Nothing) ! Incomparable_Values
```

Comment:
`Nothing` used to be there to signal "incomparable values". Do we now have two ways to signal the same thing? Does returning `Nothing` means the same thing as `Error.throw Incomparable_Values`?

Reply:
Sorry, this was my mistake. The signature should not change. Rewriting this back to `compare : T -> T -> (Ordering|Nothing)`.